<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFMPEG_WRAPPER</class>
 <widget class="QMainWindow" name="FFMPEG_WRAPPER">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1921</width>
    <height>1284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FFMPEG_WRAPPER</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="infoCrawlTab">
       <attribute name="title">
        <string>infoCrawl</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="infoCrawlGetRootButton">
            <property name="text">
             <string>Select Parent Directory...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="infoCrawlProgressBar">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="infoCrawlTableWidget">
          <property name="autoScroll">
           <bool>false</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Extension</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Number of Files</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Total Size</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Shape</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>nFrames</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>FPS</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>codec</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="vidoeCompressTab">
       <attribute name="title">
        <string>operations</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QComboBox" name="mainModeComboBox">
            <item>
             <property name="text">
              <string>--- select mode ---</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Single Video</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Folder Crawler</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mainEncodingComboBox">
            <item>
             <property name="text">
              <string>--- target encoding ---</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AVI</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MP4 (H265)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mainGrayscaleCheckBox">
            <property name="text">
             <string>Grayscale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mainLosslessCheckBox">
            <property name="text">
             <string>Lossless</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="mainQuality1Label">
                 <property name="text">
                  <string>Quality (CRF)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="mainQuality2Label">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="mainQualityCRFSlider">
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="maximum">
                <number>51</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
               <property name="tickInterval">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="mainSourceButton">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Select Source</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mainTargetButton">
            <property name="text">
             <string>Select Target</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mainConvertButton">
            <property name="text">
             <string>Convert</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
       <zorder></zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1921</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
